gammapy/maps/tests/test_axes.py

```diff
@@ -180,6 +180,26 @@ def test_up_downsample_consistency(factor):
     assert_allclose(axis.edges, axis_new.edges)
 
 
+def test_strict_downsample():
+    axis1 = MapAxis.from_bounds(
+        10.0, 2e3, 6, interp="log", name="energy_true", unit="GeV"
+    )
+    with pytest.raises(ValueError) as exc_info:
+        axis1.downsample(4)
+    assert str(exc_info.value) == "Number of energy_true bins is not divisible by 4"
+
+    axis1_strict_down = axis1.downsample(4, strict=True)
+    assert_allclose(axis1_strict_down.nbin, 0.5 * (axis1.nbin) - 1)
```

Comment:
What is this number you are checking against? It should be `(axis1.nbin)/factor + 1`, no?

Reply:
That would be `(axis1.nbin)/4 + 1 = 2.5`